charts/node/README.md

```diff
@@ -68,6 +68,31 @@ node:
       vaultKey: nodekey
 ```
 
+## Upgrade
+### From v2.x.x to v3.0.0 (⚠️ breaking changes)
+There are now separate volumes for:
+- relaychain data
+- relaychain keystore
+- parachain data
+- parachain keystore
+
+Some chart parameters have been grouped together and renamed. There are now separate sections for the following values:
+- `node.chainData`
+- `node.chainKeystore`
+- `node.collator`
+
+Common `storageClass` parameter has been moved to the corresponding separate groups mentioned above.
+
+As both the chain data and keystore can now be stored on up to 4 different volumes you may need to manually relocate the existing data to the newly created volumes.
+
+If you're running a collator node:
+- move chain keystore from `/data/chains/<chain_name>/keystore` to `/keystore`
+- move relaychain data from `/data/relay/polkadot` to `/relaychain-data/polkadot`
+- move relaychain keystore from `/data/relay/polkadot/chains/<relay_chain_name>/keystore` to `/relaychain-keystore`
+
+If you're running a non-collator node:
+- move chain keystore from `/data/chains/<chain_name>/keystore` to `/keystore`
+
```

Comment:
Let's be extra precise in the steps details, this is the verbatim of what I would write:

```
If you're running a non-collator node:
- Move keystore files from the `/data/chains/<chain_name>/keystore` in the `chain-data` volume to `/keystore` in the `chain-keystore` volume.

If you're running a collator node:
- Move keystore files from the `/data/chains/<chain_name>/keystore` in the `chain-data` volume to `/keystore` in the `chain-keystore` volume.
- Move relaychain files from the `/data/relay/polkadot` in the `chain-data` volume to `/relaychain-data/polkadot` in the `relaychain-data` volume.
- Move relaychain keystore from `/data/relay/polkadot/chains/<relay_chain_name>/keystore` in the `chain-data` volume to `/relaychain-keystore` in the `relaychain-keystore` volume
```